mlir/python/mlir/dialects/_affine_ops_ext.py

```diff
@@ -0,0 +1,54 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+try:
+    from ..ir import *
+    from ._ods_common import get_op_result_or_value as _get_op_result_or_value
+    from ._ods_common import get_op_results_or_values as _get_op_results_or_values
+except ImportError as e:
+    raise RuntimeError("Error loading imports from extension module") from e
+
+from typing import Optional, Sequence, Union
+
+
+class AffineStoreOp:
+    """Specialization for the Affine store operation."""
+
+    def __init__(
+        self,
+        value: Value,
+        memref: Union[Operation, OpView, Value],
+        map,
+        *,
+        map_operands=[],
```

Comment:
This is one of those [python gotchas](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) that actually exists in other places in our code base. Can you make this `=None` and then add something like `map = map if map is not None else []` (not necessarily that concise - typing on my phone...).

Reply:
Thanks!  In fact that map = map ...  is concise (error free).   Updated accordingly.

Reply:
same for `map_operands` 🙂

Reply:
Fixed. :)  I still kept 2 commits so the fix for map_operands is squashed into previous review commit.